docs/src/remote-wallet/ledger.md

```diff
@@ -43,6 +43,48 @@ Read more about [sending and receiving tokens](../cli/transfer-tokens.md) and
 [delegating stake](../cli/delegate-stake.md). You can use your Ledger keypair URL
 anywhere you see an option or argument that accepts a `<KEYPAIR>`.
 
+### Install the Solana Beta App
+
+For those of you that enjoy life on the edge, you can use the command-line to
+install the latest Solana Ledger app release before it has been validated by
+the Ledger team and made available via Ledger Live.  Note that because the app
+is not installed via Ledger Live, you will need to approve installation from an
+"unsafe" manager, as well as see the message, "This app is not genuine" each
+time you open the app. Once the app is available on Ledger Live, you can
+reinstall the app from there, and the message will no longer be displayed.
+
+1. Connect your Ledger device via USB and enter your pin to unlock it
+2. Download and run the Solana Ledger app installer:
+   ```text
+   curl -sSLf https://github.com/solana-labs/ledger-app-solana/releases/download/v0.2.0/install.sh | sh
+   ```
+3. When prompted, approve the "unsafe" manager on your device
+4. When prompted, approve the installation on your device
+5. An installation window appears and your device will display "Processing..."
+6. The app installation is confirmed
+
+If you encounter the following error:
```

Comment:
There's another error case if `python3-venv` isn't installed. The error message is pretty helpful on its own though, not sure if we want  to mention it?

```text
The virtual environment was not created successfully because ensurepip is not
available.  On Debian/Ubuntu systems, you need to install the python3-venv
package using the following command.

    apt-get install python3-venv

You may need to use sudo with that command.  After installing the python3-venv
package, recreate your virtual environment.

Failing command: ['/code/solana/ledger-app-solana/ledger-env/bin/python3', '-Im', 'ensurepip', '--upgrade', '--default-pip']

sh: 1836: ledger-env/bin/pip3: not found
/code/solana/ledger-app-solana/ledger-env/bin/python3: Error while finding module specification for 'ledgerblue.loadApp' (ModuleNotFoundError: No module named 'ledgerblue')
```

Reply:
Yeah, that's such a great error message. I'll leave it out until there's questions about it.